### PR TITLE
Update tenant name

### PR DESCRIPTION
**Pull request by JoMas971**
This PR must be merged after #488.

### CODE REVIEW

File: src/sdk/model/vac/ba-company.ts [hunk 1]
```diff
@@ -541,4 +545,38 @@ export class BaCompany extends Entity {
     }
     return Iland.getHttp().post(`/vac-companies/${this.uuid}/actions/request-upgrade-contract`, request);
   }
+
+  /**
+   * Update BaCompany name
+   * @param {BaCompanyUpdateRequest | BaCompanyUpdateRequestJson} updateRequest
+   * @returns {Promise<BaCompany>}
+   */
+  async updateVacCompany(updateRequest: BaCompanyUpdateRequest | BaCompanyUpdateRequestJson): Promise<BaCompany> {
```
Comment (cfsnyder) [inline src/sdk/model/vac/ba-company.ts]: why take both types?

File: src/sdk/model/vac/ba-company.ts [hunk 2]
```diff
@@ -541,4 +545,38 @@ export class BaCompany extends Entity {
...
+      return this;
+    });
+  }
+
+  /**
+   * Reset the password for a given tenant (BaCompany)
+   * @param {BaCompanyPasswordResetRequest | BaCompanyPasswordResetRequestJson} resetPasswordRequest
+   * @returns {Promise}
+   */
+  async resetVacCompanyPassword(resetPasswordRequest: BaCompanyPasswordResetRequest |
```
Comment (cfsnyder) [inline src/sdk/model/vac/ba-company.ts]: same question here?

File: src/sdk/model/vac/ba-company.ts (hunk 1 above)
Comment (JoMas971) [reply src/sdk/model/vac/ba-company.ts]: Fixed.

File: src/sdk/model/vac/ba-company.ts (hunk 2 above)
Comment (JoMas971) [reply src/sdk/model/vac/ba-company.ts]: Fixed.

**Pull request merged**